### ISSUE TITLE
Pretrained transformers require apex

### ISSUE THREAD

**Issue by emilydinan** [facebookresearch/ParlAI#1912]
**Bug description**
All of the [pre-trained transformers](https://github.com/facebookresearch/ParlAI/tree/master/parlai/zoo/pretrained_transformers) in the zoo require GPUs and apex to be installed in order to be loaded.

All of these models need to be re-saved on CPU and uploaded to the zoo. Additionally, we should have a test that verifies that these models are working properly.

**Reproduction steps**
Try loading one of the pretrained transformer models on a machine without GPUs.

**Expected behavior**
Loads.

**Logs**
```
>>> model = torch.load('/Downloads/polyranker/model', map_location=lambda cpu, _: cpu)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/edinan/anaconda3/lib/python3.6/site-packages/torch/serialization.py", line 368, in load
    return _load(f, map_location, pickle_module)
  File "/Users/edinan/anaconda3/lib/python3.6/site-packages/torch/serialization.py", line 542, in _load
    result = unpickler.load()
ModuleNotFoundError: No module named 'apex'
```

**Additional context**
See #1907.

**Comment by sravankumarlalam** [comment]
Hi, I encountered the same error...Can you please elaborate on the solution a bit. I could not understand the Additional context provided by you

**Comment by klshuster** [comment]
Hi @sravankumarlalam, I am working on a solution and will update this issue when a fix is available

**Comment by klshuster** [comment]
Fixed in #2009, will close this task when merged

**Comment by klshuster** [comment]
This issue is fixed by #2009
